src/interfaces/sdkTypes.ts

```diff
@@ -1,4 +1,6 @@
 import Decimal from 'decimal.js';
+import { OrderStatus, PositionStatus, PythData, SnxAccountType } from './subgraphTypes';
+import { S } from '@parifi/synthetix-sdk-ts/dist/index-BTlMb-Ja';
```

Comment:
Remove this. No import should be done from the dist directory

Reply:
Done